pkg/plugin/processor/builtin/unwrap_examples_test.go

```diff
@@ -0,0 +1,100 @@
+// Copyright © 2024 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package builtin
+
+import (
+	"github.com/conduitio/conduit-commons/opencdc"
+	sdk "github.com/conduitio/conduit-processor-sdk"
+	"github.com/conduitio/conduit/pkg/foundation/log"
+	"github.com/conduitio/conduit/pkg/plugin/processor/builtin/unwrap"
+)
+
+//nolint:govet // we're using a more descriptive name of example
+func ExampleUnwrapKafkaConnect() {
+	p := unwrap.NewKafkaConnectProcessor(log.Nop())
+
+	RunExample(p, example{
+		Description: `This example shows how to unwrap a Kafka Connect record.
+
+The Kafka Connect record is serialized as a JSON string in the .Payload.After field (raw data).
+The Kafka Connect record's payload will replace the OpenCDC record's payload.
+
+We also see how the key is unwrapped too. In this case, the key comes in as structured data.
+`,
+		Config: map[string]string{},
+		Have: opencdc.Record{
+			Position:  opencdc.Position("test position"),
+			Operation: opencdc.OperationCreate,
+			Metadata: opencdc.Metadata{
+				"metadata-key": "metadata-value",
+			},
+			Key: opencdc.StructuredData{
+				"payload": map[string]interface{}{
+					"id": 27,
+				},
+				"schema": map[string]interface{}{},
+			},
+			Payload: opencdc.Change{
+				After: opencdc.RawData(`{
+						"payload": {
+							"description": "test2"
+						},
+						"schema": {}
+					}`),
```

Comment:
btw, this formatting looks _very_ strange in the generated docs, I'd lose the tabs and use spaces if you ask me, also move everything to the left.
```suggestion
  "payload": {
    "description": "test2"
  },
  "schema": {}
}`),
```

Reply:
Or maybe even shove everything in a single line?

Reply:
I've moved everything to the left, since it's nicer to read in the test, and might produce a nice diff. If it's ugly in the docs, we can always change it.